packages/react-components/react-table/src/hooks/useTableFeatures.ts

```diff
@@ -45,6 +46,7 @@ export function useTableFeatures<TItem>(
     // eslint-disable-next-line @typescript-eslint/naming-convention
     columnSizing_unstable: defaultColumnSizingState,
     tableRef: React.createRef(),
+    accessibilityMenuItems: [],
```

Comment:
thinking about this a bit more... do we really need a new part of state for this? All that's used in userland is the click handler. Couldn't we just export `enableInteractiveMode` and let the user use that however they want?

We still have guidance on how we expect this interactive mode to be toggled but aren't opinionated on how to do so.

Reply:
removed